fuzzers/aflchurn/fuzzer.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Integration code for AFLChurn fuzzer."""
+
+import shutil
+import subprocess
+import os
+
+from fuzzers import utils
+
+
+# '-fsanitize-coverage=trace-pc-guard'
+def prepare_build_environment():
```

Comment:
please don't repeat this code, can use these functions from afl fuzzer by importing that module. other afl fuzzer variants do the same.

Reply:
Thanks, will try to fix that.

Reply:
Hi @inferno-chromium , I still use my own definition of `prepare_build_environment()`. Although the code is similar, they actually are quite different.
I don't want to use `trace-pc-guard`, so I don't use the `afl.prepare_build_environment()`. Even if `trace-pc-guard` is allowed, I still have to repeat all the left lines except `os.environ['FUZZER_LIB'] = '/libAFL.a'`.
So, will that be a problem that I don't call `afl.prepare_build_environment()`?